stonesoup/sensormanager/optimise.py

```diff
@@ -0,0 +1,102 @@
+from abc import abstractmethod
+from collections import defaultdict
+
+import numpy as np
+from scipy.optimize import brute, basinhopping
+
+from . import BruteForceSensorManager
+from ..base import Property
+
+
+class _OptimizeSensorManager(BruteForceSensorManager):
+
```

Comment:
I assume this gets left out of the Docs?

Reply:
Otherwise needs an explanation.

Reply:
Nope it's not included in the docs. I've updated the documentation slightly to make it clearer how the available classes work